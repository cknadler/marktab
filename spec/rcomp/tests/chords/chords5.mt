# chord shorthand syntax
(6,5:3 2,1:1)
