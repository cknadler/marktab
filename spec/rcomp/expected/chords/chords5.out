e|-1-|
B|-1-|
G|---|
D|---|
A|-3-|
E|-3-|

